<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alex R</author>
 <comment>
Copyright (C) 2010 Alex R burbulator@gmail.com

This file is part of Qvernote.

Qvernote is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

Qvernote is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with Qvernote.  If not, see http://www.gnu.org/licenses/.
 </comment>
 <class>QNotesViewClass</class>
 <widget class="QMainWindow" name="QNotesViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNotesView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>566</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pbNewNote">
          <property name="text">
           <string>New note</string>
          </property>
          <property name="icon">
           <iconset resource="Resources/Icons.qrc">
            <normaloff>:/IconAdd/general_add.png</normaloff>:/IconAdd/general_add.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDelete_notes">
    <property name="title">
     <string>Delete notes</string>
    </property>
    <addaction name="actionDelete_notes"/>
    <addaction name="actionReload"/>
   </widget>
   <addaction name="menuDelete_notes"/>
  </widget>
  <action name="actionDelete_notes">
   <property name="text">
    <string>Manage notes</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
